tests/users.test.js

```diff
@@ -1,20 +1,21 @@
-const request = require("supertest");
 const app = require("../app");
-const mongoose = require("mongoose");
-const { setupTestDatabase } = require("../tests/fixtures/db");
-
-beforeEach(setupTestDatabase);
+const request = require("supertest");
+const Users = require("../models/users");
+const { userOne } = require("./fixtures/users");
 
-afterAll(async () => {
-  await mongoose.connection.close();
+afterEach(() => {
+  jest.restoreAllMocks();
 });
 
 test("should get users", async () => {
```

Comment:
Would be good to have `describe` wrappers for each route, as there will probably be more than one test per route eventually

```suggestion
describe("GET /users", () => {
  test("should get users", async () => {
  ...
```

Reply:
good idea!